File: docs-old/datatools/codesnippet/CSharp/save-data-with-the-tableadapter-dbdirect-methods_2.cs
        private void UpdateButton_Click(object sender, EventArgs e)
        {
            Int32 newRegionID = 5;
            
            try
            {
                regionTableAdapter1.Update(newRegionID, "Updated Region Description", 5, "NorthEastern");
            }
            catch (Exception ex)
            {
                MessageBox.Show("Update Failed");
            }
            RefreshDataset();
        }

File: docs_old/datatools/codesnippet/VisualBasic/save-data-with-the-tableadapter-dbdirect-methods_2.vb
    Private Sub UpdateButton_Click() Handles UpdateButton.Click

        Dim newRegionID As Integer = 5

        Try
            RegionTableAdapter1.Update(newRegionID, "Updated Region Description", 5, "NorthEastern")

        Catch ex As Exception
            MessageBox.Show("Update Failed")
        End Try

        RefreshDataset()
    End Sub